konami/io.md

```diff
@@ -52,14 +53,9 @@
 * Product: PWB11106490000
 * Codename: "Alard"
 
-## USBIO2
+## KDE-J
```

Comment:
KDE-J just means Konami Digital Entertainment Japan, it's still called the USBIO2.